### PR TITLE
Added Child Healh Monthly Map report

### PR DESCRIPTION
**Pull request by lwyszomi**
@calellowitz @sheelio

### CODE REVIEW

File: custom/icds_reports/filters.py [hunk 1]
```diff
@@ -218,3 +220,105 @@ def options(self):
...
+    def selected(self):
+        return super(YesNoResidentFilter, self).selected or self.options[0][0]
+
+    @property
+    def options(self):
+        return [
+            ('All', 'All'),
+            ('yes', 'Yes'),
+            ('no', 'No')
+        ]
```
Comment (dimagimon) [inline custom/icds_reports/filters.py]: W292 no newline at end of file

File: custom/icds_reports/reports/__init__.py [hunk 2]
```diff
@@ -122,14 +122,16 @@ def _unformat_row(row):
         table.insert(0, [export_sheet_name])
         return [export_sheet_name, table]
 
-from custom.icds_reports.reports.reports import MPRReport, ASRReport, TableauReport
+from custom.icds_reports.reports.reports import MPRReport, ASRReport, TableauReport, ChildHealthMonthReport, \
```
Comment (dimagimon) [inline custom/icds_reports/reports/__init__.py]: F401 'ChildHealthMonthReport' imported but unused

**Comment by sheelio** [comment]
@lwyszomi 
Can you add some screenshots to this if easy to get?

**Comment by lwyszomi** [comment]
@sheelio Added screenshots:

Filters:
filters

State view:
states

District view:
districts

**Comment by sheelio** [comment]
@lwyszomi 

is possible to change the map background to not show country borders?  Are we able to zoom in directly on INdia?

**Comment by lwyszomi** [comment]
@sheelio:
new_states_map

**Comment by lwyszomi** [comment]
@sheelio I added labels to the map, new screenshots on mapbox map below:

Districts with labels:
districts_with_labels

States with labels:
states_with_lables


Popup example:
popup_on_map

**Comment by lwyszomi** [comment]
@sheelio If you want I can change map implementation and use http://datamaps.github.io/ , currently I'm using implementation which exist in HQ

**Pull request closed without merging**